requirements.txt

```diff
@@ -1,8 +1,9 @@
 black==21.6b0
 cryptography==35.0.0
 Flask==2.0.1
+flake8==4.0.1
```

Comment:
Probably don't need this, I'm guessing it's a dependency of one of the other packages we explicitly need here right?

Reply:
I only ask because I ran into situations where conflicts came about before when doing a `pip freeze` on all of the packages used here, instead of defining only the ones that are explicitly needed at letting pip handle their dependencies.

Reply:
Whoops you're right, that was added by accident! However, this requirements.txt is sort of meant for dev right? (since setup.py defines actual production requirements). If so, probably doesn't hurt to leave it on there, since `black` is already on the list (and both are required for precommit). I sort of started migrating my personal stuff from requirements.txt -> Pipfile to avoid these confusions, but that's an annoying process on the CI side.

Reply:
Your call for keep or remove, but I'd say do the same for black & flake8

Reply:
I vote remove. Keeping all the ~60 dependencies that are brought in from this file has given me various issues in the past that I would like to avoid :+1:

Reply:
Flake8 is gone, wasn't sure if you meant black too (or sphinx too? Kind of fall in the same boat of not always necessary dev tools) so just let me know!